src/vocutouts/workers.py

```diff
@@ -75,111 +85,86 @@ class TaskTransientError(Exception):
 
 
 @dramatiq.actor(queue_name="cutout", max_retries=1)
-def cutout_range(
+def cutout(
     job_id: str,
-    data_id: Dict[str, Union[str, int]],
-    ra_min: float,
-    ra_max: float,
-    dec_min: float,
-    dec_max: float,
+    data_ids: List[str],
+    stencils: List[Dict[str, Any]],
 ) -> List[Dict[str, Any]]:
-    """Implement a cutout for a specified range."""
+    """Perform a cutout.
+
+    This is a queue worker for the vo-cutouts service.  It takes a serialized
+    cutout request, converts it into a suitable in-memory format, and then
+    dispatches it to the scientific code that performs the cutout.  The
+    results are stored in a GCS bucket, and the details of the output are
+    returned as the result of the worker.
+
+    Parameters
+    ----------
+    job_id : `str`
+        The UWS job ID, used as the key for storing results.
+    data_ids : List[`str`]
```

Comment:
Just to avoid conclusion in the future, `dataset_ids` would be a much better name in terms of consistency with butler terminology.